test_opensearchpy/test_async/test_server/test_helpers/conftest.py

```diff
@@ -43,12 +43,24 @@ async def client() -> Any:
 
 @fixture(scope="function")  # type: ignore
 async def opensearch_version(client: Any) -> Any:
-    info = await client.info()
+    """
+    yields the version of the OpenSearch cluster
+    :param client:
+    :return:
+    """
```

Comment:
missing param description

Reply:
I'm also adding a pylint extension to add missing-param-doc (when a parameter is missing from the docstring) and differing-param-doc (when a parameter is different in the function definition and the docstring).